sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.institutions.controller.ts

```diff
@@ -113,8 +113,12 @@ export class ConfirmationOfEnrollmentInstitutionsController extends BaseControll
             applicationNumber:
               disbursement.studentAssessment.application.applicationNumber,
             applicationId: disbursement.studentAssessment.application.id,
-            studyStartPeriod: getISODateOnlyString(offering.studyStartDate),
-            studyEndPeriod: getISODateOnlyString(offering.studyEndDate),
+            offeringIntensity:
+              disbursement.studentAssessment.application.offeringIntensity,
+            studentNumber:
+              disbursement.studentAssessment.application.studentNumber,
+            studyStartDate: getISODateOnlyString(offering.studyStartDate),
+            studyEndDate: getISODateOnlyString(offering.studyEndDate),
```

Comment:
No conversion required for date-only values. It can be assigned as below.
```ts
studyStartDate: offering.studyStartDate,
studyEndDate: offering.studyEndDate,
```